spec/lib/efile/nj/nj1040_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
+
+describe Efile::Nj::Nj1040 do
+  let(:intake) { create(:state_file_nj_intake) }
+  let(:instance) do
+    described_class.new(
+      year: MultiTenantService.statefile.current_tax_year,
+      intake: intake
+    )
+  end
+
+  context 'when filing status is single' do
+    it "sets line 6 to 1000" do
+      instance.calculate
+      expect(instance.lines[:NJ1040_LINE_6].value).to eq(1000)
+    end
+
+    context 'when filer is older than 65' do
+      before do
+        intake.primary_birth_date = Date.new(1900, 1, 1)
```

Comment:
[pebble/dust] what are the tradeoffs of changing the value on an existing intake as opposed to re-creating an intake with the desired field in the setup?

My gut says that I would always rather recreate using the factory because otherwise you open yourself up to issues regarding state leaking between tests and test order mattering to whether it passes/fails that can get really hard to debug and spot.

Was there a reason for this approach? Otherwise I'd prefer to recreate the intake rather than modify it

Reply:
No reason! Didn't realize it was possible to define `primary_birth_date` in the `create(:state_file_nj_intake)` call. Thanks for demonstrating the pattern in your PR and advocating for it here.